package.json

```diff
@@ -33,7 +33,7 @@
   "homepage": "https://github.com/silvermine/standardization#readme",
   "dependencies": {
     "@commitlint/cli": "12.1.1",
-    "@silvermine/markdownlint-rule-indent-alignment": "github:silvermine/markdownlint-rule-indent-alignment",
+    "@silvermine/markdownlint-rule-indent-alignment": "github:silvermine/markdownlint-rule-indent-alignment#597894f7096389721a89eed91444ff5d93d400bc",
```

Comment:
I just published v0.1.0 of this: https://www.npmjs.com/package/@silvermine/markdownlint-rule-indent-alignment

Can we switch to it?